CUBRID/cubrid-testtools
# [CUBRIDQA-33] support shell and isolation test

In this pull request, we enhanced CTP to support both shell and isolation test.

How to execute test:
```
ctp.sh shell -c CTP/conf/shell.conf
ctp.sh isolation -c CTP/conf/isolation.conf
```
Both shell and isolation tests are designed to execute on multiple test nodes since they are slow on execution. The unique control node and one or more test nodes should all deploy CTP program.
And test cases should be checked out in each test nodes.  And you have to spend a little time on configuration. 
In each test node of shell test, you should set environmental variable ```init_path```, like ```export init_path=$HOME/CTP/shell/init_path```;  In each test node of isolation test, it requires like ```export CTP_HOME=$HOME/CTP```.  Actually, we expect only CTP_HOME is always used. Currently, all existing shell test cases are using ```$init_path``` .
